core/templates/services/context.service.spec.ts

```diff
@@ -213,6 +282,12 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        expect(ecs.getPageContext()).toBe('story_editor');
```

Comment:
Where do we set the context in this test?

Reply:
Ditto: spyOn in beforeEach

Reply:
Please set the context in this test and include a precheck as required.

Reply:
Done!